resolver/ng/init.sls

```diff
@@ -40,3 +40,18 @@
     - onchanges:
       - file: {{ sls }}~update-resolv.conf-file
 {% endif %}
+
+{{ sls }}~disable-networkmanager-resolvconf:
+  file.line:
+    - name: {{ resolver.ng.networkmanager.file }}
+    - after: '[main]'
+    - content: 'dns=none'
+    - mode: ensure
+    - onlyif: systemctl is-enabled NetworkManager
```

Comment:
I think we need to move this to cover whole `disable-networkmanager-resolvconf` section, because it's also needed in `service.running` (it fails if no network.manager service).
so it could be something like this:

```
{% if salt['cmd.run']('systemctl is-enabled NetworkManager') %}
{{ sls }}~disable-networkmanager-resolvconf:
...
{% endif %}
```

Reply:
Great feedback @AAbouZaid  I'll implement & test your suggestion.